### ISSUE TITLE
Won't install on Seamonkey2

### ISSUE THREAD

**Issue by cub-uanic** [binaryage/firequery#8]
Your exstention won't install on Seamonkey2, with Firebug 1.5b6 installed, saying "this is not Firefox". However, Seamonkey2 use Gecko/20091112, and AFAIK browser is based on Firefox 3.5.

**Comment by darwin** [comment]
feel free to add Seamonkey GUID into
http://github.com/darwin/firequery/blob/master/firefox/install.rdf

Please test it and if it works, please send me a patch. Thanks!

**Comment by cub-uanic** [comment]
Sorry, but I have no idea about how to do this. I don't know how to develop browser extensions. I even don't know how and where I can get Seamonkey GUID. But I can give you all needed info - just describe what you need and how I can get it for you.

Also, I'm not sure that this is good idea at all - dependency from application GUID. For example, I can use Seamonkey on Windows platform, and Iceape on Debian. Not sure, but I think that GUIDs will be different. Do you want to have all possible GUIDs in your extension?

**Comment by darwin** [comment]
GUIDs will be the same. 

I'm sorry I don't use Seamonkey and I don't have time to install and test it.

You need to specify new "required" property in install.rdf:
https://developer.mozilla.org/en/Install_manifests

Here is a list of GUID I've found with Seamonkey:
http://www.get-firefox.eu/default_025.html

Maybe you find a better docs.

compiling XPI is really easy, just run rake task

**Comment by darwin** [comment]
s/I don't have time/don't want to/  :-)

**Comment by cub-uanic** [comment]
To make trivial update you propose me learn about XPI, install ruby and rake, fork your project and prepare patches?.. You don't think that it will be simpler for you than for me? Very friendly, just nothing to say... 

Sorry, but I'm not XPI developer. I can be tester, if you want/need. But if improving _your_ software not needed for you, then it unconditionally not needed for me. You don't need free testers? Well, then good luck to you and your project.

**Comment by darwin** [comment]
alright. I'll look at it during weekend.

I understand your point, but look at it from my side. I've built this for myself and invested few more days for packaging, releasing and answering questions on this bug list, etc. But this is a bonus work I did as a volunteer. This does not mean I'm going to support every request I got as if it is a commercial product. I've never used Seamonkey and I have no plan to start using it. 

I would give you friendly recommendation. You would do better stop crying you are "not XPI developer", learn something and help yourself. Nobody cares you don't know something.
